### PR TITLE
ProtectedActionNavigationPropAPI

### PR DESCRIPTION
**Pull request by bakerCaleb**
Doc updates for API changes to support protected actions, with navigation property for authcontext

### CODE REVIEW

**Comment by dodaromike** [comment]
@bakerCaleb The changelog says: "Added the **authenticationContextId** property to unifiedrbacresourceaction resource."  But, I don't see it in the list of properties in api-reference/beta/resources/unifiedrbacresourceaction.md.

**Comment by dodaromike** [comment]
Three commits for small changes.  @Danielabom for another pass.

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -1,5 +1,40 @@
 {
   "changelog": [
+
+      {
+        "ChangeList": [
+          {
+            "Id": "9f793143-536a-4f23-acc3-54c937fccbe6",
+            "ApiChange": "Relationship",
+            "ChangedApiName": "unifiedrbacresourceaction",
```
Comment (Danielabom) [inline changelog/Microsoft.DirectoryServices.json:9]:
```suggestion
            "ChangedApiName": "authenticationContext",
```

Comment (Danielabom) [inline changelog/Microsoft.DirectoryServices.json:9]:
```suggestion
            "ChangedApiName": "authenticationContext",
```

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+            "ApiChange": "Relationship",
+            "ChangedApiName": "authenticationContext",
+            "ChangeType": "Addition",
+            "Description": "Added the **authenticationContext** relationship to the [unifiedrbacresourceaction](https://learn.microsoft.com/graph/api/resources/unifiedrbacresourceaction?view=graph-rest-beta) resource.",
+            "Target": "unifiedrbacresourceaction"
+          },
+          {
+            "Id": "8486b1f7-22b2-425b-bae7-e08c4cb7146b",
+            "ApiChange": "Property",
+            "ChangedApiName": "unifiedRbacApplication",
```
Comment (Danielabom) [inline changelog/Microsoft.DirectoryServices.json:17]:
```suggestion
            "ChangedApiName": "authenticationContextId",
```

**Comment by Danielabom** [comment]
@bakerCaleb, the build keeps failing. Please look at the errors. Thanks!

**Pull request closed without merging**